src/lib/popover/popover.component.ts

```diff
@@ -52,31 +54,41 @@ const DEFAULT_TRANSITION  = '200ms cubic-bezier(0.25, 0.8, 0.25, 1)';
 })
 export class SatPopover implements AfterViewInit {
 
-  /** Position of the popover on the x axis. */
+  /** Alignment of the popover on the horizontal axis. */
   @Input()
-  get xPosition() { return this._xPosition; }
-  set xPosition(val: SatPopoverPositionX) {
-    this._validateXPosition(val);
-    if (this._xPosition !== val) {
-      this._xPosition = val;
-      this._setPositionClasses();
+  get horizontalAlign() { return this._horizontalAlign; }
+  set horizontalAlign(val: SatPopoverHorizontalAlign) {
+    this._validateHorizontalAlign(val);
+    if (this._horizontalAlign !== val) {
+      this._horizontalAlign = val;
+      this._setAlignmentClasses();
       this._dispatchConfigNotification(new PopoverNotification(NotificationAction.REPOSITION));
     }
   }
-  private _xPosition: SatPopoverPositionX = 'center';
+  private _horizontalAlign: SatPopoverHorizontalAlign = 'center';
 
-  /** Position of the popover on the y axis. */
+  /** Alignment of the popover on the x axis. Alias for `horizontalAlign`. */
   @Input()
-  get yPosition() { return this._yPosition; }
-  set yPosition(val: SatPopoverPositionY) {
-    this._validateYPosition(val);
-    if (this._yPosition !== val) {
-      this._yPosition = val;
-      this._setPositionClasses();
+  get xAlign() { return this.horizontalAlign; }
+  set xAlign(val: SatPopoverHorizontalAlign) { this.horizontalAlign = val; }
+
+  /** Alignment of the popover on the vertical axis. */
+  @Input()
+  get verticalAlign() { return this._verticalAlign; }
+  set verticalAlign(val: SatPopoverVerticalAlign) {
+    this._validateVerticalAlign(val);
+    if (this._verticalAlign !== val) {
+      this._verticalAlign = val;
+      this._setAlignmentClasses();
```

Comment:
I need to verify if this line is really needed here. It seems that this should be covered by the positioning callbacks in the anchor and could only make the classes wrong when they're changed while the popover is open.